### DISCUSSION TITLE
How to create a random delay time?

### DISCUSSION THREAD

**Discussion by eduard-buta** [AutomaApp/automa#333] [Q&A]
Hi there. As the title suggests, I'm struggling with creating a random delay time. 

I've tried adding the random javascript function already implemented within Automa into a Delay block, without any luck. 

Is there a way to achieve this?

**Comment by Kholid060** [comment]
Right now it's not possible to create a random delay time, but in the next update, I'll enable the [reference data](https://docs.automa.site/api-reference/reference-data.html) feature in the delay block so you can use the [`$randit`](https://docs.automa.site/api-reference/reference-data.html#randint-min-max) function to generate a random delay time.
